.github/workflows/tests.yml

```diff
@@ -104,7 +102,9 @@ jobs:
         apt-get install -qy gcc make linux-headers-generic linux-image-generic openssl shim-signed
 
     - name: Install dkms
-      run: make install
+      run: |
+        sed -i -e '/echo -en "."/d' dkms.in
```

Comment:
@scaronni, why not just remove these echo lines from `dkms.in` directly? Is the goal to make this change only when testing on GitHub, but not when run locally?

Reply:
Correct, only in test.

This is to remove the printing of "progressing dots", i.e. every 3 seconds a new dot gets printed until the task is completed. Depending on the environment, the length of the dot line can of course be different. But when running it interactively on a system it's a useful indicator to check the status of the task, if it's stuck or not.

For example the Nvidia driver prints a very long line of dots when using the open source module tarball.